Content/ClosePrintSpooler.md

```diff
@@ -0,0 +1,28 @@
+# Close Print Spooler
+
+_ClosePrintSpooler_ is a sandbox setting that provided nuanced control over how sandboxed applications interact with the print spooler service.
+
+```
+   .
+   .
+   .
+   [DefaultBox]
+   ClosePrintSpooler=n
+```
+
+This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.
+
+Added as part of 0.5.4 / 5.46.0 version.
```

Comment:
```suggestion

Added as part of 0.5.4 / 5.46.0 version.

This setting can be used to prevent sandboxed applications from interacting with the print spooler service. When set to `y`, sandboxed applications will be unable to interact with the print spooler service - for example, print.
```

Reply:
I don't know - usually the "added" is "the most useless part" of the documentation
(At least, when it is that wordy).

Does it make sense to emphasize it that much by putting it first?

While the project does not PUSH for updates, it is a "reasonable expectation" that users opt to update?